warehouse/query-core/src/main/java/datawave/next/stats/StatUtil.java

```diff
@@ -13,10 +15,34 @@ public class StatUtil {
      * @return the formatted time
```

Comment:
I'd like to see the documentation here include examples of what to expect back for different durations.